yt_idv/scene_data/block_collection.py

```diff
@@ -5,6 +5,7 @@
 from yt.data_objects.data_containers import YTDataContainer
 
 from yt_idv.opengl_support import Texture3D, VertexArray, VertexAttribute
+from yt_idv.scene_data._geometry_utils import phi_normal_planes
```

Comment:
note to self: can remove the phi-normal plane calculations for this PR -- the code will exist in #64 should we want to add it back in the future...